code/packages/web/src/pages/dashboard/dashboard_pages/profile/Profile.tsx

```diff
@@ -1,3 +1,94 @@
+import Grid from '@mui/material/Grid2';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import Avatar from '@mui/material/Avatar';
+import Divider from '@mui/material/Divider';
+import Button from '@mui/material/Button';
+import Copyright from 'src/internals/components/Copyright';
+import { styled } from '@mui/system';
+
+const profileData = {
+  id: '12345',
+  username: 'johndoe',
+  email: 'john.doe@example.com',
+  firstName: 'John',
+  lastName: 'Doe',
+  company: 'Tech Solutions',
+  phone: '+1 123-456-7890',
+};
+
+const StyledCard = styled(Card)(({ theme }) => ({
+  borderRadius: theme.spacing(2),
+  boxShadow: theme.shadows[5],
+  overflow: 'hidden',
+}));
+
 export const Profile = () => {
-  return <>Profile</>;
+  return (
+    <Box sx={{ width: '100%', maxWidth: { sm: '100%', md: '800px' }, mx: 'auto', p: 3, borderRadius: 2 }}>
+      <CardContent sx={{ textAlign: 'center', mb: 3 }}>
+        <Avatar
+          alt={profileData.firstName + ' ' + profileData.lastName}
+          src="/path/to/avatar.jpg"
+          sx={{ width: 120, height: 120, margin: '0 auto', mb: 2 }}
+        />
+        <Typography component="h2" variant="h4" fontWeight="bold">
+          {profileData.firstName} {profileData.lastName}
+        </Typography>
+        <Typography variant="subtitle1" color="text.secondary">
+          {profileData.company}
+        </Typography>
+        <Button variant="contained" color="primary" sx={{ mt: 2 }}>
+          Edit Profile
+        </Button>
+      </CardContent>
+      <StyledCard sx={{ mb: 3 }}>
+        <CardContent>
+          <Typography variant="h6" color="primary" sx={{ mb: 2, fontWeight: 'medium' }}>
+            Contact Information
+          </Typography>
+          <Divider sx={{ mb: 2 }} />
+          <List disablePadding>
+            <ListItem>
+              <ListItemText primary="ID" secondary={profileData.id} />
```

Comment:
Remove this list Item. We dont want to show `id` is an internal property and doesn't have anything to the user. Showing this will expose our internal ids